model/client/common/xrpl_response.go

```diff
@@ -0,0 +1,5 @@
+package common
+
+type XRPLResponse interface {
```

Comment:
I've put this in the client package as I think it makes more sense to keep it there.

Reply:
Moved it :)